src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,8 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e :
+                    raise CallError(f'Certificate request failed: {e}')
```

Comment:
Can we please reword this to `Certificate request for final order failed: {e}` ?